tests/draw.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { MemoryModel } from "../src/memory_model";
```

Comment:
The two imports are inconsistent, and they shouldn't need to be. You can import both `MemoryModel` and `draw` from `../src/index`, as that file exports both of them. (Not sure if importing from `index.js` is special in some way.)

Reply:
This evolved into something larger. Essentially, the new import now looks like
```
import exports from "../src";
const { MemoryModel, draw } = exports;
```
which is valid syntax (and cleaner) outside of this context. However, it appears that RoughJS relies on ESM exports, which Jest has [confirmed](https://github.com/jestjs/jest/issues/9430#issuecomment-623941616) that it won't support. 

As a result, with this setup, Jest reports an error along the lines of `Unexpected token export`. This is fixed by forcing a manual resolve with Jest, as mentioned on [stackoverflow](https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library/73203803#73203803) and specifically [this comment](https://github.com/uuidjs/uuid/issues/451#issuecomment-1112328417). The updated jest config now has 
```
  moduleNameMapper: {
    // Force module roughjs to resolve with the CJS entry point, because Jest does not support package.json.exports. Elaborated in PR#15.
    "roughjs": require.resolve('roughjs'),
  },
```
Although unlikely, if Jest at some point supports ESM, then this configuration can be removed. As shown above, I left a comment alongside the config for future reference.

Reply:
Related: because `MemoryModel` is exported as a class and then dynamically imported, in order to use it as a type, I had to use `InstanceType<typeof MemoryModel>` instead of simply `MemoryModel`. See t[his stackoverflow comment](https://stackoverflow.com/questions/62059408/reactjs-and-typescript-refers-to-a-value-but-is-being-used-as-a-type-here-ts/63639280#63639280) for more elaboration.